src/components/howTos.stories.mdx

```diff
@@ -0,0 +1,35 @@
+<Meta title="How-tos" />
+
+# How-to guides
+
+You can refer to the usage and some examples below.
+
+## Enabling content share in Electron applications
+
+In an Electron application, you should create a custom screen picker to retrieve the screen's source ID and pass it to the useContentShareControls hook to start sharing your screen.
```

Comment:
Actually we do not pass the source ID to `useContentShareControls` hook right? So, should we say something like below:

"In an Electron application, you have to create a custom screen picker to retrieve the screen's source ID and pass it to the `toggleContentShare` method from `useContentShareControls` hook to start sharing your screen."